<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>345</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>313</width>
    <height>345</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>313</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="MyTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>140</width>
      <height>325</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Элементы</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="AddDelGroup">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>141</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Добавление</string>
    </property>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>46</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-10000000</number>
     </property>
     <property name="maximum">
      <number>10000000</number>
     </property>
    </widget>
    <widget class="QComboBox" name="TypeItemBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Целый</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Вещественный</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Символьный</string>
      </property>
     </item>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000000000.899999976158142</double>
     </property>
     <property name="maximum">
      <double>100000000.989999994635582</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="AddButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="DelButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Извлечь</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AddButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить в конец</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DelButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Извлечь из конца</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ListTypeBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Стек</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Очередь</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дек</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
